From 02ac53280fef79d7ed2aafa2989cd1da5c5e8d80 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Mon, 8 Jan 2018 14:23:49 +0200
Subject: [PATCH] Use icaltime_is_utc() instead of icaltimetype::is_utc

The structure member is going to be removed from libical3 and the
function is available for a long time, thus it's safe to replace
the usage in the code.

(ported from commit 7c0f42604c16fe571168ea48f4628bc920359595)
---
 src/calendar/e-cal-backend-ews-utils.c | 2 +-
 src/calendar/e-cal-backend-ews.c       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/calendar/e-cal-backend-ews-utils.c b/src/calendar/e-cal-backend-ews-utils.c
index 41d77b4..a46b1e3 100644
--- a/src/calendar/e-cal-backend-ews-utils.c
+++ b/src/calendar/e-cal-backend-ews-utils.c
@@ -346,7 +346,7 @@ ewscal_set_time (ESoapMessage *msg,
 	gchar *tz_ident = NULL;
 
 	if (with_timezone) {
-		if (t->is_utc || !t->zone || t->zone == icaltimezone_get_utc_timezone ()) {
+		if (icaltime_is_utc (*t) || !t->zone || t->zone == icaltimezone_get_utc_timezone ()) {
 			tz_ident = g_strdup ("Z");
 		} else {
 			gint offset, is_daylight, hrs, mins;
diff --git a/src/calendar/e-cal-backend-ews.c b/src/calendar/e-cal-backend-ews.c
index 50dd5ae..29e488f 100644
--- a/src/calendar/e-cal-backend-ews.c
+++ b/src/calendar/e-cal-backend-ews.c
@@ -1081,7 +1081,7 @@ e_cal_backend_ews_get_timezone_from_ical_component (ECalBackend *backend,
 			struct icaltimetype dtstart;
 
 			dtstart = icalproperty_get_dtstart (prop);
-			if (dtstart.is_utc)
+			if (icaltime_is_utc (dtstart))
 				tzid = "UTC";
 		}
 	}
-- 
2.15.1

